### PR TITLE
Remove resolve_reverse_lookup setting

### PR DESCRIPTION
**Pull request by horgh**
This basically just reverts 5b60b859d45feaa5504a196d77a5a4a318130493 which added it.

### CODE REVIEW

**Comment by ailin-nemui** [comment]
@irssi/developers

File: src/core/net-nonblock.c [hunk 1]
```diff
@@ -27,13 +27,11 @@
 
 /* nonblocking gethostbyname(), ip (IPADDR) + error (int, 0 = not error) is
    written to pipe when found PID of the resolver child is returned */
-int net_gethostbyname_nonblock(const char *addr, GIOChannel *pipe,
-			       int reverse_lookup)
+int net_gethostbyname_nonblock(const char *addr, GIOChannel *pipe)
```
Comment (dequis) [inline src/core/net-nonblock.c]: This looks API breaking, keep the param

Comment (ailin-nemui) [reply src/core/net-nonblock.c]: maybe better to break the API since the feature is gone?

Comment (dequis) [reply src/core/net-nonblock.c]: I don't think this feature is as important as the ability to do DNS resolutions at all. Callers won't really care if reverse lookup is done or not.

Comment (horgh) [reply src/core/net-nonblock.c]: Good catch, thank you! It's there again.

**Comment by dequis** [comment]
Thanks! Looks good. 

ABI version bump maybe?

**Comment by dequis** [comment]
(RESOLVED_IP_REC becoming shorter)

**Pull request merged**